app/containers/RoomHeader/RoomHeader.tsx

```diff
@@ -179,11 +192,18 @@ const Header = React.memo(
 		const handleOnPress = useCallback(() => onPress(), []);
 
 		const accessibilityLabel = useMemo(() => {
+			const iconOrStatusLabel = getRoomAccessibilityLabel({
+				isGroupChat,
+				status: status ?? statusState,
+				teamMain,
+				type: prid ? 'discussion' : type,
+				userId: roomUserId
+			});
 			if (tmid) {
-				return `${title} ${parentTitle}`;
+				return `${iconOrStatusLabel} ${title} ${parentTitle ?? ''}.`;
 			}
-			return title;
-		}, [title, parentTitle, tmid]);
+			return `${iconOrStatusLabel} ${title} ${subtitle ?? ''}.`;
+		}, [title, parentTitle, tmid, isGroupChat, status, statusState, teamMain, type, roomUserId, prid]);
```

Comment:
Create useAccessibilityLabel hook.
You won't need `useMemo` with it and you can move the logic away from this component.